lib/ravendb/documents/session/in_memory_document_session_operations.rb

```diff
@@ -0,0 +1,130 @@
+module RavenDB
+  class InMemoryDocumentSessionOperations
+    attr_reader :included_documents_by_id
+
```

Comment:
missing properties and fields see https://github.com/ravendb/ravendb-jvm-client/blob/48d6516935eb78009e000b6f72687318d7fb088f/src/main/java/net/ravendb/client/documents/session/InMemoryDocumentSessionOperations.java#L41
Among others:
- _clientSessionId 
- _clientSessionIdCounter 
- _requestExecutor
- generateDocumentKeysOnStore 
- sessionInfo
- _saveChangesOptions
- _isDisposed
- id
etc.